package.json

```diff
@@ -7,7 +7,7 @@
     "bootstrap": "shx rm -rf dist && shx mkdir dist",
     "start": "npm run build && run-p server watch:**",
     "build": "run-s bootstrap build:**",
-    "build:config": "node js/build-config.js > config/env.generated.json",
+    "build:config": "node js/build-env-config.js > config/env.generated.json",
```

Comment:
it looks like PORT is missing from `default.env` resulting in an `env.generated.json` with `ORIGIN:"https://....:undefined"`